mrmustard/lab/gates.py

```diff
@@ -797,3 +801,69 @@ def __init__(
     @property
     def Y_matrix(self):
         return gaussian.noise_Y(self.noise.value, settings.HBAR)
+
+
+class PhaseNoise(Parametrized, Transformation):
+    r"""The phase noise channel.
+
+    The phase noise channel is a non-Gaussian transformation that is equivalent to
+    a random phase rotation.
+
+    Args:
+        phase_stdev (float or List[float]): the standard deviation of the (wrapped) normal
```

Comment:
```suggestion```
Args:
    phase_stdev: the standard deviation of the (wrapped) normal distribution for the angle of the rotation
    phase_stdev_trainable: whether the phase standard deviation should be trainable or not
    phase-stdev_bounds: the upper and lower bounds for the phase's standard deviation
    modes: the single mode this gate is applied to (default [0])